README.md

```diff
@@ -53,19 +53,19 @@ Arguments:
  - jar_fpath (str, optional): Filepath to OPSIN jar file. Defaults to "opsin-cli.jar" which is distributed with py2opsin.
 
 
-## Speedup 50x from `pubchempy`
-`py2opsin` runs locally and is smaller in scope in what it provides, which makes it __dramatically__ faster at resolving identifiers. In the code block below, the call to `py2opsin` will execute ~58x faster than an equivalent call to `puchempy`:
+## Massive speedup from `pubchempy` for batch translations
+`py2opsin` runs locally and is smaller in scope in what it provides, which makes it __dramatically__ faster at resolving identifiers. In the code block below, the call to `py2opsin` will execute ~23 faster than an equivalent call to `pubchempy`:
```

Comment:
Let's get rid of the numerical comparison here - going to vary based on chemicals, machines, internet speed, etc.

Reply:
OK, fixed that up in the latest commits